mods/ts/rules/defaults.yaml

```diff
@@ -227,13 +227,21 @@
 		TargetTypes: Ground, Infantry
 	RenderSprites:
 	WithInfantryBody:
-	WithDeathAnimation:
+	WithDeathAnimation@normal:
 		DeathTypes:
 			BulletDeath: 1
 			SmallExplosionDeath: 2
-			ExplosionDeath: 3
-			InfantryExplodeDeath: 4 #not used by any warhead; TODO: check what should use this type and add it!
+			HeadshotDeath: 3 #not used by any warhead; TODO: check what should use this type and add it!
```

Comment:
No, something is getting fixed there. Our die3 did not existed in TS and before this comment, our die4 had no sequence.

TS die3 was InfantryExplode=S_BANG34. (See your tibsun.mix/local.mix/rules.ini along with http://modenc.renegadeprojects.com/InfDeath). When @reaperrr managed to sneak in said unused (infdie.shp) asset into the game, he inserted it before the last 3 used deathanims (explosion, flameguy, electro)

infdie is a headshot animation. What he did there is to restore the used death type and rename the unused one. ExplosionDeath got shifted from 3 to 4 and the other got a new name. The to-do still stands.